### PR TITLE
Sec and miner  no longer start with glowstick.

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

Sec starts with 3 glowstick, they dont need another one

## Why It's Good For The Game

Sec already starts with 3 glowstick and a flashlight.

## Changelog
:cl:
tweak: sec starts wih 3 glowstick now, like in the old times
fix: error in the description on the tula drink

/:cl:

### CODE REVIEW

**Comment by DaddyIssues98** [comment]
i will never know why the checks fail

**Comment by DaddyIssues98** [comment]
also... includes were duplicate? what is going on i dont want to fix things i want to fix my things

File: tgstation.dme [hunk 1]
```diff
@@ -3406,9 +3406,6 @@
 #include "ModularBungalow\transfervote\code\shuttle.dm"
 #include "ModularBungalow\transfervote\code\transfervote.dm"
 #include "ModularBungalow\transfervote\code\transfervote_config.dm"
-#include "ModularBungalow\Mining\items.dm"
-#include "ModularBungalow\Mining\monster.dm"
-#include "ModularBungalow\Mining\rockplanet.dm"
```
Comment (EgorDinamit) [inline tgstation.dme:3411]: ???

**Comment by EgorDinamit** [comment]
I think you should've checked the boxes' override code instead.
You can't just assign value to null here.

File: tgstation.dme (hunk 1 above)
Comment (DaddyIssues98) [reply tgstation.dme:3411]: that was duplicate, atleast is what the checks/github say, they are still include in the code

**Comment by DaddyIssues98** [comment]
> I think you should've checked the boxes' override code instead.
> You can't just assign value to null here.

there is already "null" is medipens 3 lines above, so i was assuming you can do the same

**Comment by DaddyIssues98** [comment]
dont know how to fix, samll brain moment, going to try in the future

**Pull request closed without merging**